3D/senior-csharp-test/Assets/Cache/Test/FileCacheStorageTest.cs

```diff
@@ -1,7 +1,6 @@
 using NUnit.Framework;
 using Zenject;
 
-
 [ TestFixture ]
 public class FileCacheStorageTest : ZenjectUnitTestFixture
```

Comment:
if we add a Sample test, we could add the as well? or do you think it's not necessary?

Reply:
I think we discussed about not helping the user on the testing thing (that's why I remove also the hint), he should know already OR check by himself.

However I wonder if we need to give him zenject since we need to call the FileCacheStorage constructor ourselves (can't be injected like we do for other module)

Reply:
I guess we should not include it yes, it may be confusing to leave it

Reply:
but since we added a SampleTest, I was wondering the extends of having samples in the classe